### PR TITLE
adding salsa support

### PR DESCRIPTION
**Pull request by DiegoSena**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -195,6 +195,20 @@ You can change Graphile address and port using the flags graphile-url.
 /nonodo --graphile-url http://mygraphileaddress:5034
```
Comment (fabiooshiro) [inline README.md:195]: I think it's better to remove the `/`.

Comment (fabiooshiro) [reply README.md:195]: image

It's nice to show the endpoint in the log.

./nonodo --salsa

Http Rollups for development started at http://localhost:5004
**Lambada running at http://localhost:5005**
GraphQL running at http://localhost:8080/graphql
Inspect running at http://localhost:8080/inspect/
Press Ctrl+C to stop the node

[16:42:33.621] INF nonodo: ready after=1.895389s

Comment (sandhilt) [inline README.md:195]:
```suggestion
nonodo --graphile-url http://mygraphileaddress:5034
```

**Comment by fabiooshiro** [comment]
there is no salsa log:
```bash
➜  nonodo git:(adding-salsa-support) go build
➜  nonodo git:(adding-salsa-support) ./nonodo --salsa -d
[16:47:59.133] DBG repository/input_repository.go:61 Inputs table created
[16:47:59.134] DBG repository/report_repository.go:29 Reports table created
[16:47:59.134] DBG reader/adapter_v1.go:32 NewAdapterV1
[16:47:59.134] DBG repository/report_repository.go:29 Reports table created
[16:47:59.134] DBG repository/input_repository.go:61 Inputs table created
[16:47:59.135] DBG devnet/anvil.go:94 anvil: anvil is installed
[16:47:59.135] DBG devnet/anvil.go:132 anvil: created temp dir with state file dir=/var/folders/0_/wl5k8lm94jggr2c14r0fl8n00000gn/T/1573303443
[16:47:59.237] DBG supervisor/supervisor.go:59 supervisor: worker is ready worker=anvil
[16:47:59.241] DBG supervisor/supervisor.go:59 supervisor: worker is ready worker=inputter
[16:47:59.241] DBG inputter/inputter.go:71 readPastInputs startBlockNumber=0 dappAddress=0xab7528bb862fb57e8a2bcd567a2e929a0be56a5e
[16:47:59.241] DBG supervisor/supervisor.go:59 supervisor: worker is ready worker=http
[16:47:59.241] DBG supervisor/supervisor.go:59 supervisor: worker is ready worker=http

Http Rollups for development started at http://localhost:5004
GraphQL running at http://localhost:8080/graphql
Inspect running at http://localhost:8080/inspect/
Press Ctrl+C to stop the node

[16:47:59.241] INF nonodo/main.go:485 nonodo: ready after=1.178407791s
```

**Review by fabiooshiro — APPROVED** [review]

**Pull request merged**